interpreter/terminal_interface/magic_commands.py

```diff
@@ -74,33 +90,153 @@ def handle_debug(self, arguments=None):
     else:
         display_markdown_message("> Unknown argument to debug command.")
 
+
 def handle_reset(self, arguments):
     self.reset()
     display_markdown_message("> Reset Done")
 
+
 def default_handle(self, arguments):
     display_markdown_message("> Unknown command")
     handle_help(self,arguments)
 
 def handle_save_message(self, json_path):
     if json_path == "":
-      json_path = "messages.json"
+        json_path = "messages.json"
     if not json_path.endswith(".json"):
-      json_path += ".json"
+        json_path += ".json"
     with open(json_path, 'w') as f:
-      json.dump(self.messages, f, indent=2)
+        json.dump(self.messages, f, indent=2)
+
+    display_markdown_message(
+        f"> messages json export to {os.path.abspath(json_path)}")
 
-    display_markdown_message(f"> messages json export to {os.path.abspath(json_path)}")
 
 def handle_load_message(self, json_path):
     if json_path == "":
-      json_path = "messages.json"
+        json_path = "messages.json"
     if not json_path.endswith(".json"):
-      json_path += ".json"
+        json_path += ".json"
     with open(json_path, 'r') as f:
-      self.messages = json.load(f)
+        self.messages = json.load(f)
+
+    display_markdown_message(
+        f"> messages json loaded from {os.path.abspath(json_path)}")
+
+def handle_container_upload(self,type=None, *args):
```

Comment:
We can address this in a separate PR afterward, but I want to leave a note here to refer back to.

From a user experience perspective, I think this should just be a polymorphic function that lets me either run `%upload`, `%upload misc/tests/OpenInterpreter-Notebook-Mode.gif`, or `%upload misc/tests/data` and figures out if I have referenced a file or directory and acts accordingly instead of requiring the `file` or `folder` keyword.